<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditarTurma</class>
 <widget class="QDialog" name="EditarTurma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="cursoLabel">
         <property name="text">
          <string>Curso</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="cursoLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="moduloLabel">
         <property name="text">
          <string>Módulo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="moduloLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="statusDaTurmaLabel">
         <property name="text">
          <string>Status da Turma</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="statusDaTurmaLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="tabelaSelecionaAlunos"/>
       </item>
       <item>
        <widget class="QTableWidget" name="tabelaSelecionaProfessores"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="botaoCancelar">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoConfirmar">
         <property name="text">
          <string>Confirmar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoEditarHorarios">
         <property name="text">
          <string>Editar 
 Horários</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
